### PR TITLE
Hacktoberfest 2021

### PR DESCRIPTION
**Pull request by kpodemski**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | hacktoberfest announcement
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

File: _data/authors.yml [hunk 1]
```diff
@@ -198,9 +198,9 @@ mdekker:
 
 kpodemski:
     display_name: Krystian Podemski
-    role: Community Member
+    role: Tech Evangelist @ PrestaShop
```
Comment (matks) [inline _data/authors.yml:201]: Time changes 😄

File: news/_posts/2021-10-01-hacktoberfest-2021.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+icon: icon-github-alt
+tags: [community, contribution, github, contribute]
+---
+
+Hacktoberfest is a month-long celebration of open-source software that happens online every October, sponsored by Digital Ocean, the American cloud hosting company [DigitalOcean](https://www.digitalocean.com). People are encouraged to engage and create at least 4 pull requests for open-source projects on GitHub.
+
+If you want to contribute to open-source projects but don’t know where to start, Hacktoberfest might be the perfect opportunity for you.
+
+
+![Hacktoberfest 2021](/assets/images/2021/10/hacktoberfest.svg)
```
Comment (matks) [inline news/_posts/2021-10-01-hacktoberfest-2021.md:17]: Should we add a mention to say this image does not belong to us? Or does Hacktoberfest clearly mention we can reuse without quoting author?

**Review by matks — COMMENTED** [review]
Good writing 👍 

We should use a script to add the `hacktoberfest` label to all our repos to opt-in. I have a set of scripts that do that kind of stuff on https://github.com/matks/prestashop-repos-bulk-editor . With a few modifications it should be doable.

File: _data/authors.yml (hunk 1 above)
Comment (kpodemski) [reply _data/authors.yml:201]: yeah 😁

File: news/_posts/2021-10-01-hacktoberfest-2021.md (hunk 2 above)
Comment (kpodemski) [reply news/_posts/2021-10-01-hacktoberfest-2021.md:17]: I got it from there:
https://hacktoberfest.digitalocean.com/brandguidelines

We can use it to promote event

**Review by matks — APPROVED** [review]
I approve the PR. It can be published on 1st of October.

**Review by PierreRambaud — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Comment by kpodemski** [comment]
@PrestaShop/prestashop-maintainers can we publish it? I have no rights to merge in this repo

**Review by jolelievre — APPROVED** [review]

**Pull request merged**